### DISCUSSION TITLE
Custom environments

### DISCUSSION THREAD

**Discussion by michielswaanen** [Infisical/infisical#105] [Feature Request]
Currently, these are hardcoded environments: Development, Staging, Production, and Testing.

It would be helpful to have a custom environment option.
Thus, being able to remove or add other options.

**Comment by vmatsiiako** [comment]
Hey @MichielS-1848709! I agree, this is super important. We actually have an issue for that (https://github.com/Infisical/infisical/issues/31). I think George ended up working on a different issue with integrations, so this one is currently not handled. 

I will try to prioritize that in what I'm doing. There is just so much engineering right now that we can't keep up with everything :D 
If you want to contribute even a little bit - this would be really appreciated. We can help you with understanding the codebase quickly! Alternatively, if you know anyone who could be interested in contributing, please propose it to them :)

**Reply by michielswaanen** [reply]
I understand, I'm really busy myself, but I'm willing to contribute!

**Reply by vmatsiiako** [reply]
This would be so amazing! Thank you so much! Feel free to join our community Slack with any questions: https://join.slack.com/t/infisical-users/shared_invite/zt-1kdbk07ro-RtoyEt_9E~fyzGo_xQYP6g

Also, would you want to set up a super quick pairing session to guide you through the codebase? No pressure though!

**Reply by michielswaanen** [reply]
For sure, however, I'm currently not available! I'm looking through the codebase myself now.
